L1Trigger/TrackerDTC/interface/Setup.h

```diff
@@ -81,21 +85,84 @@ namespace trackerDTC {
     GlobalPoint stubPos(const TTStubRef& ttStubRef) const;
     // empty trackerDTC EDProduct
     TTDTC ttDTC() const { return TTDTC(numRegions_, numOverlappingRegions_, numDTCsPerRegion_); }
+    // checks if stub collection is considered forming a reconstructable track 
+    bool reconstructable(const std::vector<TTStubRef>& ttStubRefs) const;
+    // checks if tracking particle is selected for efficiency measurements
+    bool useForAlgEff(const TrackingParticle& tp) const;
+    //
+    bool useForReconstructable(const TrackingParticle& tp) const { return tpSelectorLoose_(tp); }
+    //
+    int layerId(const TTStubRef& ttStubRef) const;
+    //
+    int barrel(const TTStubRef& ttStubRef) const;
+    //
+    int psModule(const TTStubRef& ttStubRef) const;
+    //
+    TTBV layerMap(const std::vector<int>& ints) const;
+    //
+    TTBV layerMap(const TTBV& hitPattern, const std::vector<int>& ints) const;
+    //
+    std::vector<int> layerMap(const TTBV& hitPattern, const TTBV& ttBV) const;
+    //
+    std::vector<int> layerMap(const TTBV& ttBV) const;
+    //
```

Comment:
Suggest grouping dPhi & dZ on neighbouring lines, with comment "Stub uncertainties projected onto (phi,z), with components added linearly (simpler)" And grouping v0 and v1 with comment "Stub uncertainties squared projected onto (phi,z)".

Reply:
done

Reply:
Could you please add the more detailed comment suggested above?